frontend/src/__tests__/cypress/cypress/pages/clusterSettings.ts

```diff
@@ -17,77 +17,77 @@ class ClusterSettings {
   }
 
   findSubmitButton() {
-    return cy.get('[data-id="submit-cluster-settings"]');
+    return cy.findByTestId('submit-cluster-settings');
```

Comment:
@christianvogt I have replaced `get` with `findByTestId` , since `get` searches from the root element , is that fine?

Reply:
The old code did this because the attribute was named `data-id` instead of `data-testid`. The use of `get` was likely an oversight, however it made no difference since it wasn't a chained command.

The new change looks correct.